ansible/library/announce_routes.py

```diff
@@ -138,6 +162,124 @@ def get_uplink_router_as_path(uplink_router_type, spine_asn):
     return default_route_as_path
 
 
+# Generate prefixs of route
+def generate_prefixs(subnet_size, ip_base, offset):
+    next_ip = get_new_ip(ip_base, offset)
+    prefixlen_v4 = (32 - int(math.log(subnet_size, 2)))
+
+    prefix = "{}/{}".format(next_ip, prefixlen_v4)
+    octets = [int(octet_str) for octet_str in str(next_ip).split(".")]
+    prefix_v6 = "20%02X:%02X%02X:0:%02X::/64" % (octets[0], octets[1], octets[2], octets[3])
```

Comment:
How about use string format (py3 style)?
And we we cannot convert the ip to prefix directly, i guess ipaddress lib can do it?